src/Microsoft.VisualStudio.Editors/PropPages/PackagePropPage.vb

```diff
@@ -539,6 +634,10 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             ' TODO: New help keyword
             Return HelpKeywords.VBProjPropAssemblyInfo
         End Function
+
+        Private Sub PackageIconUrlWarning_TextChanged(sender As Object, e As EventArgs) Handles PackageIconUrlWarning.TextChanged
```

Comment:
Can this empty event handler be removed?

Reply:
Yep!